.github/workflows/sdk-nodejs.yml

```diff
@@ -0,0 +1,38 @@
+name: Node.js SDK
+on:
+  push:
+    branches: [main]
+    paths:
```

Comment:
This makes sense to me in general but I think there might still be value in re-running SDK tests whenever there's an engine change, just to make sure the engine didn't break the sdk somehow.

So maybe we should also include changes to the engine code in here.

Also, maybe we should include changes to `internal/mage/sdk/nodejs.go`?

Reply:
This comment confirmed [a hunch](https://github.com/dagger/dagger/pull/3897#issuecomment-1323917409) - thank you!